src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {string, node, oneOf} from 'prop-types';
+import style from './Thumbnail.st.css';
+import Check from 'wix-ui-icons-common/Check';
+import {Thumbnail as CoreThumbnail, ThumbnailProps as CoreThumbnailProps} from 'wix-ui-core/Thumbnail';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {Text} from '../Text';
+
+export interface ThumbnailProps extends CoreThumbnailProps {
+  /** image of the thumbnail */
+  image?: React.ReactElement<any>;
+  /** title of the thumbnail */
+  title: string;
+  /** description of the thumbnail */
+  description?: string;
+}
+
+const StyledThumbnail = withStylable<CoreThumbnailProps>(
```

Comment:
Don't use `withStylable`, it passes irelevant props to the extended core component.
Do something like this:
https://github.com/wix/wix-ui-backoffice/blob/3d370563d9ab792eefe5f1b8d4fc5ce3df5916ee/src/components/Button/Button.tsx#L26